src/main/groovy/com/github/lkishalmi/gradle/gatling/GatlingPlugin.groovy

```diff
@@ -56,8 +56,9 @@ class GatlingPlugin implements Plugin<Project> {
 
         project.sourceSets {
             gatling {
-                scala.srcDirs 'src/gatling/scala'
-                resources.srcDirs 'src/gatling/resources'
+                scala.srcDirs gatlingExtension.simulationsDir()
```

Comment:
Without adding '=' between srcDirs and the arg, the arg is considered as an addition as used below, however the default dir like scala is never inteded to be used.